opensrp-reveal/src/main/java/org/smartregister/reveal/model/TaskDetails.java

```diff
@@ -385,11 +385,12 @@ public void setGroupedTaskCodes(String groupedTaskCodeStatusString) {
         this.groupedTaskCodes = stringBuilder.toString();
     }
 
-    public boolean isNonResidential() {
-        return getFamilyMemberNames() != null && getFamilyMemberNames().equalsIgnoreCase("NR");
+    public boolean isResidential() {
+        return getFamilyMemberNames() != null && !getFamilyMemberNames().equalsIgnoreCase("NR");
     }
 
-    public int getPendingTasksCount(){
+    public int getPendingTasksCount() {
+        if (getTaskCount() == null) return 1;
```

Comment:
Not sure I get the logic here. why not set it to zero?

Reply:
taskCount is null for non-grouped tasks. in that case there's only 1 task so returning 1 here.

Reply:
will add javadocs